library/alpine

```diff
@@ -1,143 +1,61 @@
-Architectures: arm64v8, arm32v6, ppc64le, s390x, i386, amd64
-arm64v8-GitFetch: refs/heads/rootfs/library-3.9/aarch64
-arm64v8-GitCommit: c1ce50c570682dc6c28b326293c3a3b1591468b5
-arm64v8-Directory: versions/library-3.9/aarch64
-arm32v6-GitFetch: refs/heads/rootfs/library-3.9/armhf
-arm32v6-GitCommit: ccde8c20ca748efe5e8894e31679f1ef0dc3905d
-arm32v6-Directory: versions/library-3.9/armhf
-ppc64le-GitFetch: refs/heads/rootfs/library-3.9/ppc64le
-ppc64le-GitCommit: 4066de09e4f02eb605e59903d2f6be07dd06db93
-ppc64le-Directory: versions/library-3.9/ppc64le
-s390x-GitFetch: refs/heads/rootfs/library-3.9/s390x
-s390x-GitCommit: 969001ac645d66d00c884e8c69c8f4bd01645039
-s390x-Directory: versions/library-3.9/s390x
-i386-GitFetch: refs/heads/rootfs/library-3.9/x86
-i386-GitCommit: 5be41819190491bb616c4ca83c424e3e519bbbe4
-i386-Directory: versions/library-3.9/x86
-amd64-GitFetch: refs/heads/rootfs/library-3.9/x86_64
-amd64-GitCommit: c4f4c7a6e14d6efeb9a160da464717e03d2cc3ee
-amd64-Directory: versions/library-3.9/x86_64
+Maintainers: Natanael Copa <ncopa@alpinelinux.org> (@ncopa)
+GitRepo: https://github.com/alpinelinux/docker-alpine.git
 
 Tags: edge
-Architectures: arm64v8, arm32v6, ppc64le, s390x, i386, amd64
-arm64v8-GitFetch: refs/heads/rootfs/library-edge/aarch64
-arm64v8-GitCommit: c4f57192a21b4cccb746381ccfdcde4773ddf32a
-arm64v8-Directory: versions/library-edge/aarch64
-arm32v6-GitFetch: refs/heads/rootfs/library-edge/armhf
-arm32v6-GitCommit: 189b95028d338a928ec5ffa5ac54e338c38b7ac3
-arm32v6-Directory: versions/library-edge/armhf
-ppc64le-GitFetch: refs/heads/rootfs/library-edge/ppc64le
-ppc64le-GitCommit: e3c3856f14158dabf3e4bbe46d0df20e793cbd32
-ppc64le-Directory: versions/library-edge/ppc64le
-s390x-GitFetch: refs/heads/rootfs/library-edge/s390x
-s390x-GitCommit: c4307b8ffb4ccfb2096ec567980c3495ff4e6f94
-s390x-Directory: versions/library-edge/s390x
-i386-GitFetch: refs/heads/rootfs/library-edge/x86
-i386-GitCommit: 4f4a0f7cec98cda5baad4d22f5b24d068c244ce1
-i386-Directory: versions/library-edge/x86
-amd64-GitFetch: refs/heads/rootfs/library-edge/x86_64
-amd64-GitCommit: f98a67bfa97056988402850223740741b41551d3
-amd64-Directory: versions/library-edge/x86_64
-
-Tags: 3.1
-Architectures: amd64
-amd64-GitFetch: refs/heads/rootfs/library-3.1
-amd64-GitCommit: bfaa70b8f29be1f55ec9f34a16965535109ba482
-amd64-Directory: versions/library-3.1
-
-Tags: 3.2
-Architectures: amd64
-amd64-GitFetch: refs/heads/rootfs/library-3.2
-amd64-GitCommit: fa5398cb6bb54f0093b2ad3180cf3d903afe0a7a
-amd64-Directory: versions/library-3.2
-
-Tags: 3.3
-Architectures: amd64
-amd64-GitFetch: refs/heads/rootfs/library-3.3
-amd64-GitCommit: 7532f6996facee837af283f5259ad1d071b9bd4a
-amd64-Directory: versions/library-3.3
-
-Tags: 3.4
-Architectures: amd64
-amd64-GitFetch: refs/heads/rootfs/library-3.4
-amd64-GitCommit: 3943f4dedc41b7c6144610d3e99ad0f7083c2138
-amd64-Directory: versions/library-3.4
+Architecture: arm64v8, arm32v6, arm32v7, ppc64le, s390x, i386, amd64
```

Comment:
These need to be `Architectures`. :wink:

Reply:
good catch. will fix